test cases/frameworks/37 gir cpp/meson.build

```diff
@@ -5,6 +5,12 @@ if not gi.found()
   error('MESON_SKIP_TEST gobject-introspection not found.')
 endif
 
+if host_machine.system() == 'cygwin'
+  # FIXME: g-ir-scanner seems broken on cygwin:
+  # ERROR: can't resolve libraries to shared libraries: foo++
```

Comment:
Do we have any idea why this is the case BTW? Maybe it should be reported at least...

Reply:
From a very quick look I think they don't search libs with `cyg` prefix.

Reply:
gitlab.gnome.org is down ATM, will report later.